application-analytics-ui/src/main/resources/Analytics/Code/Macros/VelocityMacros.xml

```diff
@@ -38,25 +38,24 @@
   <hidden>true</hidden>
   <content>{{velocity output='false'}}
 #macro (getDate)
-  #set ($dashbord = $doc.getObject('Analytics.Code.AnalyticsDashboardClass'))
-  ## A dashboard will have a date inside and that date will take precedence over the date set by the macro.
-  #if ($dashbord)
-    #set ($startDate = $dashbord.getProperty('startDate'))
-    #set ($endDate = $dashbord.getProperty('endDate'))
+  ## A dashboard will have the ability to add parameters to the url to filter all the macros in the page.
```

Comment:
`If the analytics filter is used on a dashboard page, this could add filter parameters in the URL and these have priority over what is set at the macro level.`